### PR TITLE
updated versions

### PR DESCRIPTION
**Pull request by jalogisch**
This includes some version updates and adjustments in the documentation.

It does also re-introduce the elasticsearch matrix

### CODE REVIEW

File: pages/installation/operating_system_packages.rst [hunk 1]
```diff
@@ -23,11 +23,13 @@ Prerequisites
 
 Make sure to install and configure the following software before installing and starting any Graylog services:
 
-* Java (>= 8)
+* Java ( 8 or 11 )
```
Comment (dennisoelkers) [inline pages/installation/operating_system_packages.rst]: I think it was better before, you can also run the server with 9 or 10.

File: pages/installation/os/ubuntu.rst [hunk 2]
```diff
@@ -39,7 +39,7 @@ The last step is to enable MongoDB during the operating system's startup::
 Elasticsearch
 -------------
 
-Graylog can be used with Elasticsearch 6.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/deb.html>`__::
+Graylog can be used with Elasticsearch 6.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/6.8/deb.html>`__::
```
Comment (dennisoelkers) [inline pages/installation/os/ubuntu.rst]: You can also use `https://www.elastic.co/guide/en/elasticsearch/reference/6.x/deb.html` afaics, so we do not have to change the url for 6.9ff.

File: pages/configuration/elasticsearch.rst [hunk 3]
```diff
@@ -14,9 +14,27 @@ Elasticsearch versions
...
+2.0.0            2.3.1
+2.0.1-2.0.3      2.3.2
+2.1.0-2.1.3      2.3.5
+2.2.0-2.2.3      2.4.4
+2.3.x-2.4.x      2.4.x, 5.6.x
+2.5.x            2.4.x, 5.6.x, 6.8.x
+3.0-3.1          5.6.x, 6.8.x
+===============  =====================
+
+.. caution:: Graylog 3.x **does not** work with Elasticsearch 7.x!
```
Comment (dennisoelkers) [inline pages/configuration/elasticsearch.rst:33]: This line is included twice now.

**Review by dennisoelkers — CHANGES_REQUESTED** [review]
Added some remarks, but great work!

File: pages/installation/operating_system_packages.rst (hunk 1 above)
Comment (jalogisch) [reply pages/installation/operating_system_packages.rst]: ok, wasn't aware of that. My information: Java 8 until the last commits that make it compatible with 11 ...

**Review by dennisoelkers — APPROVED** [review]

**Pull request merged**